spring-cloud-dataflow-server/README.adoc

```diff
@@ -163,7 +166,11 @@ The `test.docker.compose.paths` property accepts comma separated list of docker
 
 ----
 ./mvnw clean test-compile integration-test -pl spring-cloud-dataflow-server -Pfailsafe -Dgroups="docker-compose" \
-   -Dtest.docker.compose.paths=../src/docker-compose/docker-compose.yml,../src/docker-compose/docker-compose-dood.yml,../src/docker-compose/docker-compose-prometheus.yml \
+   -Dtest.docker.compose.paths=../src/docker-compose/docker-compose.yml,\
```

Comment:
To make this work on my linux box (ubuntu) I had to surround the value of `-Dtest.docker.compose.paths` with double quotes otherwise maven thought the second argument was a profile.  I'll add the quotes where necessary